frameworks/PHP/phalcon/public/index-micro.php

```diff
@@ -96,8 +105,9 @@
         ));
 
     });
-    $request = new Phalcon\Http\Request();
-    $app->handle($request->getURI());
+    
+    $url = $_REQUEST['_url'] ?? '/';
+    $app->handle($url);
```

Comment:
```php
$app->handle($_SERVER['REQUEST_URI']);
```
seems simpler

Reply:
It is, except that it only works when the app is running from root; from a subdirectory, the suggested change has to be made (which works for both, but trivial for the tests).